continuous_integration/docker/hadoop/_print_logs.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -x
```

Comment:
When does this one get run?

I would include the name of each log file before its contents.

Reply:
> When does this one get run?

This is from test.yaml, a github workflow. It is the last line.

```yaml
      - name: Print logs for debugging
        if: always()
        run: ./continuous_integration/docker/hadoop/print_logs.sh
```

Reply:
> I would include the name of each log file before its contents.

Do you mean that you would do an echo manually before each line? That is reasonable, and perhaps more reliable - I think `set -x` has led to some weird timings where some logs show up before/after etc.

Reply:
I removed `set -x` and added colorized print statements on what commands are run.